drivers/timer/sam0_rtc_timer.c

```diff
@@ -102,10 +110,21 @@ static inline void rtc_sync(void)
 static uint32_t rtc_count(void)
 {
 #ifdef RTC_READREQ_RREQ
-	RTC0->READREQ.reg = RTC_READREQ_RREQ;
-#endif
+	u32_t ret;
+
+	rtc_sync();
+	if ((RTC0->READREQ.reg & RTC_READREQ_RCONT) == 0) {
+		RTC0->READREQ.reg |= RTC_READREQ_RCONT;
```

Comment:
Ditto.  Optionally, put both line 117 and 118 into a single `|=`

Reply:
I do not think you can combine these into one operation.  The datasheet does appear to indicate that RCONT must be written before the RREQ bit.
_For the continuous read mode, RREQ bit is required to be set once the RCONT bit is set._

I think in my experimentation, I attempted to combine them.  If I recall, it does not work reliably if you do.

Overall, the entire RCONT functionality is poorly documented.

Reply:
I should also mention these changes only affect the SAM D.  I do not presently have SAM E hardware to do testing of the RCONT function for that device.  I reviewed the SAM E datasheet and believe the RCONT functionality may be different.
  
So, these changes are targeting the SAM D only at the moment.